src/vmm/src/arch/aarch64/regs.rs

```diff
@@ -419,22 +400,13 @@ pub fn save_core_registers(vcpu: &VcpuFd, state: &mut Vec<Aarch64Register>) -> R
 ///
 /// * `vcpu` - Structure for the VCPU that holds the VCPU's fd.
 /// * `state` - Input/Output vector of registers states.
-pub fn save_system_registers(vcpu: &VcpuFd, state: &mut Vec<Aarch64Register>) -> Result<()> {
+pub fn save_all_registers(vcpu: &VcpuFd, state: &mut Vec<Aarch64Register>) -> Result<()> {
     // Call KVM_GET_REG_LIST to get all registers available to the guest. For ArmV8 there are
     // less than 500 registers.
     let mut reg_list = RegList::new(500).map_err(Error::Fam)?;
```

Comment:
This is out of scope, but are we still sure there are less than 500 registers reported by `KVM_GET_REG_LIST`? How will we detect when it is no longer the case?

Reply:
That is actually not a bad test. Perhaps we should create an issue, if we don't want to address it right now.

Reply:
Created an issue: #3768